.github/workflows/linux-eic-shell.yml

```diff
@@ -264,17 +271,19 @@ jobs:
         network_types: "none"
         setup: install/setup.sh
         run: |
-          mkdir -p geo
-          dd_web_display --output geo/${DETECTOR}.root ${DETECTOR_PATH}/${DETECTOR}.xml
+          geoConverter -compact2tgeo -input ${DETECTOR_PATH}/${DETECTOR}_${{ matrix.detector_config }}.xml -output ${{ matrix.detector_config }}.root
     - uses: actions/upload-artifact@v3
       with:
-        name: epic.root
-        path: geo/
+        name: tgeo
+        path: ${{ matrix.detector_config }}.root
         if-no-files-found: error
 
   dump-constants:
     runs-on: ubuntu-latest
     needs: build-gcc
+    strategy:
+      matrix:
+        detector_config: [epic_arches, epic_brycecanyon]
```

Comment:
any reason we don't use `matrix: ${{ fromJson(needs.list-detector-configs.outputs.configs) }}` here too?

Reply:
We could indeed do this here. At this point I'm trying to limit the number of matrix jobs due to the maximum of 200 simultaneous jobs in GitHub (which we routinely hit) and the practical matter of the interface being hard to navigate (I'm working on addressing that using workflow_call). Do you think it would be of use to get for example drich_only_constants.out and/or toml?

Reply:
My bad, I hadn't realized this would get all the detectors, rather than just the two overall configs.